.github/workflows/check.yml

```diff
@@ -56,31 +120,78 @@ jobs:
         env:
           POSTGRES_PASSWORD: postgres
     steps:
-      - uses: actions/checkout@v3
-      - name: Set up JDK 8
-        uses: actions/setup-java@v3
-        with:
-          java-version: 8
-          distribution: 'temurin'
-          cache: "gradle"
-      - name: Connect
-        run: psql -h 127.0.0.1 -p 5432 -U postgres -d postgres -c "\l"
-        env:
-          PGPASSWORD: postgres
-      - name: Create database
-        run: psql -h 127.0.0.1 -p 5432 -U postgres -d postgres -c "create database ci_test;"
-        env:
-          PGPASSWORD: postgres
-      - name: Build with testing
-        run: ./gradlew --stacktrace :embulk-output-postgresql:check
+    - uses: actions/checkout@v4
+    - name: Set up JDK 8
+      uses: actions/setup-java@v4
+      with:
+        java-version: 8
+        distribution: 'zulu'
+        cache: "gradle"
+    - name: Connect
+      run: psql -h 127.0.0.1 -p 5432 -U postgres -d postgres -c "\l"
+      env:
+        PGPASSWORD: postgres
+    - name: Create database
+      run: psql -h 127.0.0.1 -p 5432 -U postgres -d postgres -c "create database ci_test;"
+      env:
+        PGPASSWORD: postgres
+    - name: Build with testing
+      run: ./gradlew --stacktrace :embulk-output-postgresql:check
+      env:
+        _JAVA_OPTIONS: "-Xmx2048m -Xms512m"
+        EMBULK_OUTPUT_POSTGRESQL_TEST_CONFIG: "${{ github.workspace }}/ci/postgresql.yml"
+    - uses: actions/upload-artifact@v4
+      if: always()
+      with:
+        name: postgresql9_4
+        path: embulk-output-postgresql/build/reports/tests/test
+  # PostgreSQL 14 and later, raise the exception "The authentication type 10 is not supported."
+  # Use PostgreSQL 13 at this time.
```

Comment:
```suggestion
  # Testing with PostgreSQL 13 here, not with PostgreSQL 14 or later, because
  # it raises an exception: "The authentication type 10 is not supported."
  # TODO: Start testing with PostgreSQL 14 or later.
```